### DISCUSSION TITLE
malformed output in when running webpack dev server

### DISCUSSION THREAD

**Discussion by dagda1** [vanilla-extract-css/vanilla-extract#594] [Q&A]
Can anyone suggest why I might see output like this when I am running webpack dev server:

>  !*** css ../../node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[9].use[1]!../../node_modules/virtual-resource-loader/dist/virtual-resource-loader.cjs.js?{"source":"I3Jvb3QgewogIGJvcmRlcjogMXB4IHNvbGlkIGdyZWVuOwp9Ci5nbG9iYWxfcGFyZW50X18xajcycDR6MCB7CiAgYm9yZGVyOiAxcHggc29saWQgcmVkOwogIGJhY2tncm91bmQ6IGdyZXk7CiAgY29sb3I6IHdoaXRlOwogIGZvbnQtd2VpZ2h0OiBib2xkOwogIG92ZXJmbG93OiBoaWRkZW47CiAgcmVzaXplOiBib3RoOwogIHdpZHRoOiAyMDBweDsKICBoZWlnaHQ6IDIwMHB4OwogIGRpc3BsYXk6IGZsZXg7CiAganVzdGlmeS1jb250ZW50OiBjZW50ZXI7CiAgYWxpZ24taXRlbXM6IGNlbnRlcjsKfQouZ2xvYmFsX2NoaWxkX18xajcycDR6MSB7CiAgYm9yZGVyOiAxcHggc29saWQgYmx1ZTsKICBiYWNrZ3JvdW5kOiBncmV5OwogIGNvbG9yOiBibGFjazsKICBmb250LXdlaWdodDogYm9sZDsKICBvdmVyZmxvdzogaGlkZGVuOwogIHJlc2l6ZTogYm90aDsKICB3aWR0aDogMjAwcHg7CiAgaGVpZ2h0OiAyMDBweDsKICBkaXNwbGF5OiBmbGV4OwogIGp1c3RpZnktY29udGVudDogY2VudGVyOwogIGFsaWduLWl0ZW1zOiBjZW50ZXI7CiAgYmFja2dyb3VuZC1jb2xvcjogd2hpdGU7Cn0="}!../../node_modules/@vanilla-extract/webpack-plugin/extracted/dist/vanilla-extract-webpack-plugin-extracted.cjs.js ***

This looks like babel output that has not been transpiled.

I am using:

```
    "@vanilla-extract/babel-plugin": "^1.1.4",
    "@vanilla-extract/css": "^1.6.8",
    "@vanilla-extract/esbuild-plugin": "^2.0.2",
    "@vanilla-extract/sprinkles": "^1.3.3",
    "@vanilla-extract/webpack-plugin": "^2.1.5",
    
```

**Comment by litewarp** [comment]
Possibly related to #374
